.gitignore

```diff
@@ -5,6 +5,9 @@ npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
 
+# add git commits (SHA1) to ignore with git blame, i.e. mass refactoring.
+.git-blame-ignore-revs
```

Comment:
Dette er nytt for meg. Skal ignore-revs-fila ikke committes?

Reply:
Jeg tenker det er best. Ikke alle bruker den, og jeg bruker den bare med en vscode-extension. Det er bare en måte å ignorere bestemte commits når man ser tilbake i historien med git blame. F.eks. bruker jeg "git-lens" i vscode for å se hvem som sist commita på en fil/linje/hunk, men jeg får ikke så mye ut av det når historien stopper på prettier-refaktoreringen fra for 18mnd siden.

Reply:
Det er jo ganske stilig. Det hadde jo vært nice om alle kunne fått tilgang til den rikere historikken.